### PR TITLE
Added declarations for 'draft-convert'.

### PR DESCRIPTION
**Pull request by guicapanema**
This is my first shot at doing both typings and GitHub pull requests, so I'm sorry in advance if there's any problems.

Added type declarations for the 'draft-convert' package which I needed to use in a project. All types are `any` for now, but if this works out I'll try and add specific types later on.

### CODE REVIEW

File: types/draft-convert/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export function convertFromHTML(...args: any[]): any;
+
+export function convertToHTML(...args: any[]): any;
+
+export function parseHTML(html: any): any;
+
+export namespace convertFromHTML {
```
Comment (andy-ms) [inline types/draft-convert/index.d.ts]: Did you use `dts-gen`? This looks like a bug; every function has an empty prototype by default, but that shouldn't need to be declared.

**Comment by andy-ms** [comment]
You don't have to add types to DefinitelyTyped to begin using them. See the [README](https://github.com/DefinitelyTyped/DefinitelyTyped#test) for how to add a type definition to your project.
At this stage of the types, I would recommend adding the types to your project, and submitting them to DefinitelyTyped once you have more complete definitions.

File: types/draft-convert/index.d.ts (hunk 1 above)
Comment (guicapanema) [inline types/draft-convert/index.d.ts]: Yes, I did! Not sure why that was created...

**Comment by guicapanema** [comment]
OK! I'll try and specify this better!

**Comment by sheetalkamat** [comment]
@guicapanema I am closing this PR and please update or create new PR when you have the types ready.

**Pull request closed without merging**